open-ce/docker_build.py

```diff
@@ -140,8 +154,8 @@ def build_in_container(image_name, output_folder, arg_strings):
     if os.path.isdir(LOCAL_FILES_PATH):
         _copy_to_container(LOCAL_FILES_PATH, HOME_PATH, container_name)
 
-    _start_container(container_name)
 
```

Comment:
looks like an extra line was added here that's not needed.

Reply:
Corrected.